Automattic/woocommerce-payments
# Deleted account status, base free from WCPay settings and migrated few components from WCPay settings to overview page.

Derived from: #1412 
Context: https://github.com/Automattic/woocommerce-payments/issues/1412#issuecomment-849894424

#### Changes proposed in this Pull Request
- Deleted account status and base fee from settings
- Migrated `ExpirationBar`, `ExpirationDescription`, and `LearnMoreLink` from WCPay settings (account-fee-settings) to the overview page.










#### Testing instructions





- switch to `migrate/account-fees-and-tests`.
- Goto WCPay settings.
- Account status and Base fee should not render.
- Goto Overview page.
- Pass dummy data (Ref. https://github.com/Automattic/woocommerce-payments/pull/1966/files#discussion_r642142580)
- It should show the expiration bar and description for promo.

#### After migration:
![image](https://user-images.githubusercontent.com/15019298/120123167-f43b1780-c1ca-11eb-9675-1523fff334b9.png)

![image](https://user-images.githubusercontent.com/15019298/120120375-70792f00-c1ba-11eb-9a6a-eb54c3c449f7.png)

![image](https://user-images.githubusercontent.com/15019298/120120387-838bff00-c1ba-11eb-99a4-0162d6f0ddf6.png)


-------------------
